From 533feadd0808ef763c82f5d49ed249797a215ea5 Mon Sep 17 00:00:00 2001
From: Nicolas Boulenguez <nicolas@debian.org>
Date: Mon, 28 Dec 2020 23:37:26 +0100
Subject: [PATCH 02/21] Fix a variable substition in debian/control

debian/control.cross.in contains this line:
  for @target@ and @target@ is not your native platform.
Currently, sed only replaces the first @target@.
---
 debian/rules | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/debian/rules b/debian/rules
index a8d1c40..af3cc28 100755
--- a/debian/rules
+++ b/debian/rules
@@ -626,7 +626,7 @@ stamps/control: debian/control.in $(if $(TARGET),debian/control.cross.in) $(if $
 ifneq (,$(TARGET))
 	sed -n -e 's/@dpkg_dev@/$(DPKG_DEV)/;/^Package: binutils-for-build$$/q;p' \
 		< debian/control.in > debian/control
-	sed -e "s/@target@/$$(echo -n $(TARGET) | sed s/_/-/g)/" \
+	sed -e "s/@target@/$$(echo -n $(TARGET) | sed s/_/-/g)/g" \
 		-e "s/@host_archs@/any/" \
 		-e 's/@binutils_alt_triplet@//' \
 		< debian/control.cross.in >> debian/control
-- 
2.30.1

